### DISCUSSION TITLE
Homepage LXC script down

### DISCUSSION THREAD

**Discussion by rcolonie** [tteck/Proxmox#897] [Help Needed] [answered]
Anyone else getting a 404 on the Homepage LXC .sh file? Seems to be gone now that I need it, too bad.

**Comment by tteck** [comment, marked as answer]
Oops... change v5 to v4 in the command. Sorry for the goof.

**Reply by rcolonie** [reply]
Thanks, did the trick. Awesome!

**Reply by tteck** [reply]
Fixed
